src/scss/components/_logosSection.scss

```diff
@@ -114,7 +115,7 @@
       border: var(--grey-5) solid 0.1rem;
     }
 
-    @include breakpoint(md) {
+    @include breakpoint(lg) {
```

Comment:
there is an issue between 1024px to 1439px,
you can fix it by changing the breakpoint "lg" to "md"